Yosemite/Yosemite/Stores/ProductStore.swift

```diff
@@ -516,18 +516,13 @@ private extension ProductStore {
     func generateProductDescription(siteID: Int64,
                                     name: String,
                                     features: String,
-                                    languageCode: String,
                                     completion: @escaping (Result<String, Error>) -> Void) {
         let prompt = [
-            "Your task is to help a marketing team create a description for a retail website of a product based on the " +
-            "properties delimited by triple backticks at the end.",
+            "Write a description for a product with title \(name) and features: \(features).",
             "Some requirements for the description are:",
-            "- Perform in-depth keyword research for the product, and include as many keywords in the description as possible. " +
-            "Do not output the list of keywords",
```

Comment:
I'm not sure why Woo AI doesn't include any keyword research, I got this from searching about how to improve SEO for an eCommerce product description. From my testing, ChatGPT is quite good at keyword research. Including the right keywords can help with SEO with potentially higher traffic.

Reply:
Restored the keyword research in a0254c68f9995e3788d979148d2bbfbfca3da971 - I updated the prompt to ensure the keywords are in the expected language and are included in sentences:

> Perform in-depth keyword research relating to the product in the same language of the product title, and use them in your sentences without listing them out